### DISCUSSION TITLE
Pull-requests search returns incomplete results

### DISCUSSION THREAD

**Discussion by park66665** [community/community#23418] [Pull Requests]
<p>I currently am looking into old pull requests of tgstation/tgstation repo. For example, this query should return every PR that has been merged 2019 May 16 through May 20: <a href="https://github.com/tgstation/tgstation/pulls?q=is%3Apr+merged%3A2019-05-16..2019-05-20" class="inline-onebox" rel="noopener nofollow ugc">Pull requests · tgstation/tgstation · GitHub</a> And it looks like the query does return matching PRs.</p>
<p>But, for example, the search result has omitted this PR for me, which has been merged on 2019 May 18: <a href="https://github.com/tgstation/tgstation/pull/43952" class="inline-onebox" rel="noopener nofollow ugc">Reworks AI backup battery system slightly, and allows (un)bolting while on backup at heavy power cost by zxaber · Pull Request #43952 · tgstation/tgstation · GitHub</a></p>
<p>I wish my query returned every PRs that matches the criteria…</p>

**Comment by vgrl** [comment, marked as answer]
<p>Hey <a class="mention" href="/park66665">@park66665</a>—welcome to the Community!</p>
<p>This type of problem is usually seen when the repository search index is out of sync, and will often resolve itself within 24 hours (as a result of a background reindexing job for the repository having completed successfully).</p>
<p>I’ve taken a look at your example (thanks for providing that!) and it appears that the issue is no longer occurring. If you are still facing this problem, I recommend opening a ticket with <a href="https://support.github.com/contact">private support</a>—they’ll be able to run through additional debugging steps with you, and manually trigger a repository search indexing job if needed.</p>
<p>Cheers, Vanessa.</p>

**Comment by park66665** [comment]
<p>Thanks for letting me know how and why it happened. I was concerned by this issue because I worked as a maintainer at a project related to the project I provided as an example. (I since resigned due to health issues)</p>
<p>Sorry for late reply. It is great to hear that the problem is resolved now.</p>
<p>Cheers,</p>
<p>~park246824</p>

**Comment by arthurlm44** [comment]
Is this repo re-indexing thing still as-is? My CI system depends on the list-associated-pull-requests feature, and I am getting an empty set as a return value from the API.
